<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>750</width><height>500</height></rect>
  </property>

  <property name="windowTitle">
   <string>Approve Loans</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">

    <!-- Title -->
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text"><string>Pending Loan Requests</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
      <property name="font">
       <font><bold>true</bold><pointsize>18</pointsize></font>
      </property>
     </widget>
    </item>

    <!-- Table -->
    <item>
     <widget class="QTableWidget" name="tableLoans"/>
    </item>

    <!-- Buttons Row -->
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="btnApprove">
        <property name="text"><string>Approve Selected Loan</string></property>
        <property name="minimumHeight"><number>40</number></property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="btnReject">
        <property name="text"><string>Reject Loan</string></property>
        <property name="minimumHeight"><number>40</number></property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- Back -->
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text"><string>Back</string></property>
      <property name="minimumHeight"><number>40</number></property>
     </widget>
    </item>

   </layout>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
